highfidelity/hifi
# Photobooth Improvements: Using Tablet & Ability to Rotate Model

This PR contains my work on getting the Photobooth to use the tablet API and adding support to rotate the model in photobooth:

- tablet button that toggles on/off for the photobooth
- using tablet event bridge
- slider bar UI that rotates model clockwise and anti-clockwise

Note: currenly there is a bug on tablet event handling which make slider bar UI works funny. It is however working great when you have tablet in toolbar mode so we recommend using that.

Test Plan:
1) Turn on toolbar tablet mode in desktop mode. (Settings > General... > Check Destop Tablet Becomes Toolbar
2) Run photoboothApp.js
3) Confirms there is now a Photobooth button on the tool bar, clicking that will bring up the UI screen and a photobooth set up will appear in world.
4) Try all functionalities and make sure it works ok.
5) Click the Photobooth button will remove all entities from Photobooth. Same happens when you end the script from Running Script.
